### DISCUSSION TITLE
Can't change the basic indoex.sql

### DISCUSSION THREAD

**Discussion by Alef15** [sqlpage/SQLPage#487] [General]
Good evening everyone,

My problem seems really basic but as I'm a noob in programming, it's normal that I hit a wall from my first attempts... Finally, with all shame, I'll still explain my situation: I'm trying to place a index.sql of my own so that http://127.0.0.1:8080/ displays it but it never works!
I try in different places, I either get a 404 error or the "it works" screen and nothing else...
So what is the basic stupidity that I am committing when I thought I had followed the tutorial to the letter?
Thanks in advance. Regards.

**Comment by lovasoa** [comment]
Hello! What does the "It works" page say?  It should say


> In order to get started, visit SQLPage's website. You can replace this page's contents by creating a file named index.sql in the folder where sqlpage is running (current working directory: /some/directory).

The directory you are looking for is the directory from this message.

**Comment by Alef15** [comment]
(current working directory: C:\SQLPage).
But, whatever I do, nothing works!
I've try to put an index.sql in C:\SQLPage but nothing changes.

**Comment by Alef15** [comment]
Something that amaze me is the "invisibility" of the standard index.sql... can't find it... May be it's normal, I do not know...

**Comment by Alef15** [comment]
But if I try http://127.0.0.1:8080/SQLPage-main/index.sql where I've change "It works!" as "It works baby!" int the index.sql present in this directory C:\SQLPage\SQLPage-main... At last, I have this working!
May be the light is at the end of the tunnel after all...

**Comment by Alef15** [comment]
Another question: is there something special to do with VS Code in order to have SQLPage syntax take in charge?

**Reply by lovasoa** [reply]
SQLPage uses the standard SQL syntax, so the default vscode syntax highlighting for sql works.
